### DISCUSSION TITLE
Need high resolution areas

### DISCUSSION THREAD

**Discussion by dchristopherfennell** [pelicanmapping/osgearth#2292] [Support Q&A]
Hello,

Is it possible to get high resolution data for the following?

- Meck Island
- Wake Island
- Kwajalein Atoll

The data we purchased does not have high resolution for these areas.

Thanks for any help with this.

DC

**Comment by plevy** [comment]
Just looking around at that area on maps, both Google and Bing are using the Maxar imagery.  Does Maxar make it easy to purchase their imagery, not really.  You can start here: https://www.maxar.com/sales-inquiries or you might get there quicker with a reseller like this one: https://geospatial.com/contact-us/

It seems like most of the resellers who made it easier to drag out an area of interest and get a quote are no longer doing that service or were folded into Maxar.

**Reply by dchristopherfennell** [reply]
I was able to request high resolution data for Kauai, Hawaii a few years ago. I was hoping that I can still do that for the locations listed above.

**Reply by plevy** [reply]
Yes, US states and territories have a few options for no-cost imagery.  Those areas you listed are as I said, mainly available through Maxar.  Can you send me an email to plevy@pelicanmapping.com or reply with yours and we can figure it the options.
